src/components/TeaList.stories.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import GlobalStyles from './GlobalStyles'
+import { addDecorator } from '@storybook/react'
+import styled from 'styled-components'
+import TeaList from './TeaList'
+import MapPinIcon from '../icons/MapPinIcon.svg'
+
+addDecorator((storyFn) => (
+  <>
+    <GlobalStyles />
+    {storyFn()}
+  </>
+))
+
+export default {
+  title: 'TeaList',
+  component: TeaList,
+}
+
+export const withExampleData = () => (
+  <Li key={'1'}>
+    <H2>{'Vietnamese Oolong'}</H2>
+    <Img src={MapPinIcon} alt="" />
+    <H3>Hier gekauft: {'Connewitzer Teeladen, Leipzig'}</H3>
+    <P>
+      {
+        'Duftiges Aroma von Orchidee. Subtiles, lang anhaltendes Bukett von Veilchen. Tee mit edlem Charakter.'
+      }
+    </P>
+    <Line />
+  </Li>
+)
+
+const Li = styled.li`
```

Comment:
Da du in deinem anderen js-Dokumenten auch immer eher die Syntax HTML-Element+Styled hast (z. B. HeaderStyled), würde ich hier bei der Benennung auch diesem Muster folgen. Also ListItemStyled, H2Styled o.Ä. :)